client/client-multi/client-c/src/kaa/platform-impl/cc32xx/logger.c

```diff
@@ -24,6 +24,7 @@
 
 void ext_write_log(FILE * sink, const char * buffer, size_t message_size)
 {
+    (void)message_size;
     if (!buffer || !sink) {
```

Comment:
That may not be necessary -- that may change nothing. But feel free to file a ticket if you think it does.